honeybee_schema/radiance/_base.py

```diff
@@ -0,0 +1,38 @@
+"""Base classs used by radiance schema objects."""
```

Comment:
```suggestion
"""Base class used by radiance schema objects."""
```

@saeranv, I suggested to @chriswmackey too to install this extension on vscode to avoid spelling issues like this one: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

It has been very helpful for me personally.

Reply:
Cool, installed.